deepchem/models/tensorgraph/layers.py

```diff
@@ -3071,3 +3071,190 @@ def none_tensors(self):
 
   def set_tensors(self, tensor):
     self.out_tensor, self.betas = tensor
+
+
+class ANIFeat(Layer):
```

Comment:
Could you add a unit test for `ANIFeat`?

Reply:
I couldn't find any tests that @miaecle used originally - do you know where they are by chance (if there are any)? This might be a pretty hard featurization to write a test for, since presumably you'd also want to test the radial and the angular components separately first.

Reply:
A simple unit test that checks `ANIFeat.create_tensor()` can be invoked without error would be good to start with :-)

Reply:
@proteneer Sorry for that but I only wrote overfitting test for ANI regressor in `deepchem/models/tests/test_overfit.py`. Second @rbharath on writing some simple tests.

Reply:
My finite difference tests invoke this implicitly :)